File: dart/easy/findTargetElemByCount.dart
class FindTargetElemByCount {
  static String result(List<String> arr, int amount) {
    var result = arr.where((item) => item.length == amount);
    if (result.length > 0) {
      return result.elementAt(0);
    } else {
      return 'nothing found';
    }
  }
}
// O(n)

File: ts/easy/findTargetElemByCount.ts
export const findTargetElemByCount = (
  arr: string[],
  target: number,
): string | undefined => {
  return arr.find((item) => item.length === target);
};
console.log(findTargetElemByCount(['Mark', 'John', 'Anna', 'Maria'], 5)); //  ['Mark', 'John', 'Anna']
// O(n)
